JSON_files/gear.JSON

```diff
@@ -0,0 +1,421 @@
+{
+  "armour": {
+    "Power Armour": {
+      "armour_value": {
+        "standard": 19,
```

Comment:
these look like quality modifiers, so maybe rename them to `armour_quality_value`? or just `armour_quality`?